server/schema/groups/createGroup.js

```diff
@@ -0,0 +1,36 @@
+const schema = {
+  required: ['group_name', 'confidential', 'creator', 'members'],
```

Comment:
creator needs not be send. We can decode the username from token itself.

Reply:
resolved